src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/WebRequestSession.cs

```diff
@@ -13,6 +14,20 @@ namespace Microsoft.PowerShell.Commands
     /// </summary>
     public class WebRequestSession
     {
+        private bool _changed;
```

Comment:
You could divide using #region; #endregion like in WebRequestPSCmdlet.Common.cs
You could move the getter setters to the top and sort them

Reply:
My concern with such changes is it will cause conflicts if others are working on this file too. Maybe once it's closer to getting accepted.

Reply:
You are right, we can move stuff in a follow up PR after we merge this PR